test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Please remove this file and next time make sure that you add any file you want to be ignored to `.gitignore` file before you push any changes.

![image](https://user-images.githubusercontent.com/68796232/158474638-3b8c2520-3a05-4179-b6bb-baca70af0363.png)

Reply:
I added it at the end to **.gitignore** because I forgot to save **test.md** changes at first. should I remove it and create it again?